### DISCUSSION TITLE
How to customize the color theme in the booksettings?

### DISCUSSION THREAD

**Discussion by hjfzbj** [Kareadita/Kavita#3471] [Ideas]
### Idea Description

![image](https://github.com/user-attachments/assets/fd8f76e1-3a90-4d16-afd8-ac52446cf75d)
Can these options be configured?

### Idea Category

Feature Enhancement

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
The book reader doesn't have a theming system but theoretically you can override anything with the theming system. What exactly are you asking?

**Reply by hjfzbj** [reply]
I have seen other systems that usually have a custom button to add custom color configurations. There seem to be only four fixed color configuration buttons here. So my suggestion is to add a custom button to this place

**Reply by scare376** [reply]
It's still not entirely clear what you are asking for. How do you want to be able to change these? Can you give an example of how another system does it that you are looking for Kavita to do? Just saying add a custom button tells us virtually nothing as to what you want the custom button to do.

**Reply by hjfzbj** [reply]
Thank you for your reply, such as, calibre , under the colors tab, there are several default color themes that come with the system, as the picture below,
![image](https://github.com/user-attachments/assets/fbe09259-9e53-4c20-b1ba-451094f77cbd)

**but it has a 'new scheme' button**, when clicking this button, its function is as follows

![image](https://github.com/user-attachments/assets/0c262410-e4b2-4c69-939d-0cbba00b6ec7)

Its function is that, in addition to selecting the system's built-in color themes, you can also add custom color schemes for yourself

**Reply by majora2007** [reply]
The problem is that the Dark/Paper/White aren't colors, they are themes. What you're asking for is a Book Theming system (which is something I have discussed as wanting to bring to Kavita). I would suggest renaming and reworking your Discussion title to indicate it and thus get more traction with the community.

**Reply by hjfzbj** [reply]
Thanks, I understand now.
